### PR TITLE
Add PDF Password Support

### PR DESCRIPTION
**Pull request by benbahrenburg**
Awesome project.

I've adding basic password support that allows the project to load password protected PDFs.

Thought you and others might find this useful.

### CODE REVIEW

File: Sources/Classes/PDFViewController.swift [hunk 1]
```diff
@@ -79,7 +79,7 @@ public final class PDFViewController: UIViewController {
     func print() {
         guard UIPrintInteractionController.isPrintingAvailable() else { return }
         guard UIPrintInteractionController.canPrintURL(document.fileURL) else { return }
-
```
Comment (ranunez) [inline Sources/Classes/PDFViewController.swift]: Theres a space missing between `{r`

**Comment by ranunez** [comment]
This looks great! It seems like theres some repetitiveness going on with the separate 
`public init(fileURL: NSURL)`
`public init(fileURL: NSURL, password: String)`

perhaps these two functions can be combined into a single 
`public init(fileURL: NSURL, password: String? = nil)`

File: Sources/Classes/PDFViewController.swift [hunk 2]
```diff
@@ -79,7 +79,7 @@ public final class PDFViewController: UIViewController {
     func print() {
         guard UIPrintInteractionController.isPrintingAvailable() else { return }
         guard UIPrintInteractionController.canPrintURL(document.fileURL) else { return }
-        
+        guard document.password == nil else {return }
```
Comment (ranunez) [inline Sources/Classes/PDFViewController.swift]: Theres a space missing between `{r`

Comment (benbahrenburg) [inline Sources/Classes/PDFViewController.swift]: I updated the formatting.

Comment (benbahrenburg) [inline Sources/Classes/PDFViewController.swift]: Combined init functions. Password is defaulted to String? = nil if any non nil value is provided we implement the standard unlock procedures

**Comment by ranunez** [comment]
Great work @benbahrenburg thanks for your contribution!

**Pull request merged**